.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: ["master"]
```

Comment:
You can try 
```yaml
on:
  push:
    branches: ["master"]
  pull_request:
    branches: ["deploy-gcp"]
```
if you want to test it on this branch. After that, you need to delete those line you just created.